core/include/usModuleSettings.h

```diff
@@ -43,11 +45,15 @@ US_BEGIN_NAMESPACE
  * - \e US_AUTOLOAD_PATHS A ':' (Unix) or ';' (Windows) separated list of paths
  *   from which modules should be auto-loaded.
  *
+ * \deprecated Use FrameworkFactory::newFramework(std::map<std::string, std::string> configuration) to configure the framework.
```

Comment:
Ok, understood. I will steer clear of "org.osgi".

I will change "org.osgi.framework.storage" to, minimally, replace "org.osgi" with "org.cppmicroservices".